src/schema/nmdc.yaml

```diff
@@ -3039,6 +3040,21 @@ slots:
       description: The conductivity measurement was 0.017 millisiemens per centimeter.
     range: QuantityValue
 
+  infiltrations:
+    annotations:
+      expected_value:
+        tag: expected_value
+        value: first infiltration time;second infiltration time;notes
+    description: The amount of time it takes to accomplish the first and second infiltration activities, and any details, issues, or context needed to understand the infiltration activity.
+    examples:
+      - value: 40 minutes;50 minutes;2017-08-24
+    aliases:
+      - infiltration_1
+      - infiltration_2
+      - infiltration_notes
+    multivalued: true
```

Comment:
Discussed this with @mslarae13 today and we had some thoughts on changing the structure. 
First, it will be okay to leave out `infiltration_notes` - it's there in the EMSL metadata submission spreadsheet, but we couldn't find any examples of anyone actually filling it out. This means that all the infiltration information will now just be the same type (time durations) instead of duration+string
Second, there could be any number of infiltrations. So what we really want is a slot holding as many `hh:mm:ss` values as the user wants to enter. Which I think means using a multivalued slot as you originally suggested is still a good idea? https://github.com/microbiomedata/nmdc-schema/issues/1338

Reply:
as to the last part - I don't believe this is going to be suggested to MIxS